Chat_App/Chat_App/UI/MessageTableViewCell.swift

```diff
@@ -0,0 +1,310 @@
+//
+//  MessageTableViewCell.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 01.05.23.
+//
+
+import UIKit
+
+final class MessageTableViewCell: UITableViewCell {
+
+    private let messageLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = MessageLabelConstants.numberOfLines
+        return label
+    }()
+
+    private let dateLabel: UILabel = {
+        let label = UILabel()
+        label.text = DateLabelConstants.labelText
+        label.font = UIFont.systemFont(ofSize: DateLabelConstants.fontSize)
+        return label
+    }()
+
+    private let chatBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = ChatBubbleConstants.cornerRadius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = ChatBubbleConstants.backgroundColor
+        return bubble
+    }()
+
+    private let middleBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = MiddleBubbleConstants.cornerRadius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = MiddleBubbleConstants.backgroundColor
+        return bubble
+    }()
+
+    private let smallBubble: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = SmallBubbleConstants.cornerRadius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = SmallBubbleConstants.backgroundColor
+        return bubble
+    }()
+
+    static let reuseIdentifier = "MessageTableViewCell"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func configure(with item: String, indexPath: IndexPath) {
```

Comment:
You should pass the model for presentation to your cell.
For instance:
```suggestion
    enum BubblePosition { case left, right }
    struct BubbleMessage { 
        let position: BubblePosition
        let message: String
    }
```